app/javascript/packs/dashboardTagsDisableUnchangedButtons.js

```diff
@@ -1,35 +1,36 @@
-document.getElementById('follows_update_form').addEventListener('submit', checkChanged);
+document
```

Comment:
Aside from changing `let`s to `const`s, there was some other linting that was corrected by the precommit run.